varnish-sys/src/vcl/processor.rs

```diff
@@ -310,7 +318,7 @@ impl<'c, 'f> FetchFilters<'c, 'f> {
         if self.find_position::<T>().is_none() {
             let instance = Box::new(new_vfp::<T>());
             unsafe {
-                ffi::VRT_AddVFP(self.ctx, instance.as_ref());
+                ffi::VRT_AddFilter(self.ctx, instance.as_ref(), ptr::null());
```

Comment:
are you just changing these to the new function? what if its for before v8?

Reply:
The `Filter` variants have been introduced in 7.2, if memory serves, so this covers all the supported version, save for 6.0

Reply:
if it does not cover all versions (6 is a version), you should introduce a new flag for this and just do conditional call here